go/libkb/kbsig.go

```diff
@@ -329,6 +349,26 @@ func (arg ProofMetadata) ToJSON(m MetaContext) (ret *jsonw.Wrapper, err error) {
 	ret.SetKey("seqno", jsonw.NewInt64(int64(seqno)))
 	ret.SetKey("prev", prev)
 
+	// If this is a standard user link, arg.Me will be provided. It is not
+	// provided during a provisionee's KEX and for team sigs, where it is
+	// overridden. It is expected to be provided by PerUserKey and
+	// Stellar proofs as well.
+	var hPrevInfo HPrevInfo
+	if arg.Me != nil {
```

Comment:
As far as I can tell `arg.Me` is only `nil` during a KEX. Is this correct?

Reply:
They're also nullable for PerUserKeyProof and StellarProof types but I'm not sure why (they're functions that take in *User instead of methods on User later in kbsig.go)

Reply:
lots of other parts of this code that fills in signature bodies requires non-`nil` arg.Me

Reply:
I wanted to make sure the only case where it is nil is during KEX so I can handle all the cases

Reply:
I just wanted to make sure that KEX was the only case where `arg.Me=nil`, so I could cover all the cases

Reply:
Shouldn't the logic go the other way... To always use HprevInfoOverride if provided? And if not fallback to arg.Me.  And if not, then return an error?

Reply:
Override is a bad name, it used to actually override before I put the multisig handling into sigchain_bump. But now exactly one of arg.Me and hPrevInfoOverride should be provided. I'll change the name and error if both or neither are provided.